### ISSUE TITLE
0.12 HEREDOC markers require following empty line

### ISSUE THREAD

**Issue by alexsomesan** [hashicorp/terraform#20541]
### Terraform Version
```
cea81d921a5b
```

### Terraform Configuration Files
```hcl
data = {
  one =<<EOF
"${filebase64("/test-fixtures/data")}"
EOF
  }
```

### Debug Output

```
        Error: Missing attribute separator: On /var/folders/zf/33sr_43n343f_wls_994vkkm0000gn/T/tf-test568793107/main.tf line 8: Expected a newline or comma to mark the beginning of the next attribute.

        State: kubernetes_secret.test:
          ID = default/tf-acc-test-gen-zlc2n
          data.% = 1
          data.one = "dGZwbGFuAv+N/5sDAQEEUGxhbgH/nAABCQEERGlmZgH/ngABBk1vZHVsZQH/rAABBVN0YXRlAf+uAAEEVmFycwH/igABB1RhcmdldHMB/4gAARBUZXJyYWZvcm1WZXJzaW9uAQwAAQ9Qcm92aWRlclNIQTI1NnMB/8IAAQdCYWNrZW5kAf+yAAEHRGVzdHJveQECAAAAH/+dAwEBBERpZmYB/54AAQEBB01vZHVsZXMB/6oAAAAm/6kCAQEXW10qdGVycmFmb3JtLk1vZHVsZURpZmYB/6oAAf+gAAAx/58DAQL/oAABAwEEUGF0aAH/iAABCVJlc291cmNlcwH/qAABB0Rlc3Ryb3kBAgAAABb/hwIBAQhbXXN0cmluZwH/iAABDAAAM/+nBAEBIm1hcFtzdHJpbmddKnRlcnJhZm9ybS5JbnN0YW5jZURpZmYB/6gAAQwB/6IAAFj/oQMBAv+iAAEFAQpBdHRyaWJ1dGVzAf+mAAEHRGVzdHJveQECAAEORGVzdHJveURlcG9zZWQBAgABDkRlc3Ryb3lUYWludGVkAQIAAQRNZXRhAf+KAAAAN/+lBAEBJm1hcFtzdHJpbmddKnRlcnJhZm9ybS5SZXNvdXJjZUF0dHJEaWZmAf+mAAEMAf+kAABv/6MDAQL/pAABCAEDT2xkAQwAAQNOZXcBDAABC05ld0NvbXB1dGVkAQI
```

### Expected Behavior
The HEREDOC expression should correctly interpolate the multiline base64 block.

### Actual Behavior
The above error is thrown about a parsing error.

### Steps to Reproduce
Apply above mentioned configuration.

/cc: @apparentlymart

**Comment by pdefreitas** [comment]
I have the same issue in a tfvars file on 0.11

**Comment by mildwonkey** [comment]
confirmed that this has been fixed, likely by #20715.
